docs/knowledgebase/solution-unable-to-apply-paramters-to-an-existing-storageclass-object.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Solution - Unable To Apply Parameters To An Existing StorageClass Object"
+linkTitle: "Solution - Unable To Apply Parameters To An Existing StorageClass Object"
+---
+
+## Issue
+
+You have created a custom `StorageClass` object in your Ondat cluster but you are experiencing an issue where you cannot interactively edit or patch the said `StorageClass` object and apply a custom `StorageClass` parameter to enable specific Ondat [features](/docs/concepts/labels/).
```

Comment:
object **or** apply a custom..

Reply:
I think "to" is the better choice here. I will change it.